sormas-api/src/main/java/de/symeda/sormas/api/InfrastructureDataReferenceDto.java

```diff
@@ -32,6 +32,7 @@ protected InfrastructureDataReferenceDto(String uuid, String caption, String ext
 		this.externalId = externalId;
 	}
 
+	// todo shouldn't this be also a list then?
```

Comment:
Should it? To my understanding a lab message could point to multiple facilities, but a facility is still identified by a single external id

Reply:
Correct. I think we can currently leave it this way. I added a remark for reference DTOs to #9389 in case we want to change something there.

Reply:
I also think this should not be a list, especially not for #8949. Shouldn't this comment just be removed?

Reply:
removed